internal/generator/vector/output/loki/userpass.go

```diff
@@ -0,0 +1,20 @@
+package loki
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/output/security"
+)
+
+type UserNamePass security.UserNamePass
+
+func (up UserNamePass) Name() string {
+	return "elasticsearchUsernamePasswordTemplate"
```

Comment:
please change to `lokiUsernamePasswordTemplate`

Reply:
done